certified-connectors/My Acclaro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,13061 @@
+            "type": "string",
+            "required": true,
+            "description": "The unique identifier assigned to the Order, i.e. Order ID.",
+            "x-ms-summary": "orderid"
+          },
+          {
+            "name": "fileid",
+            "in": "path",
+            "x-ms-url-encoding": "single",
+            "type": "string",
+            "required": true,
+            "description": "the unique identifier assigned to a source or target file, i.e. the file ID.",
+            "x-ms-summary": "fileid"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "string"
+            }
+          },
+          "401": {
+            "description": "401",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "success": {
+                  "type": "boolean",
+                  "description": "success"
+                },
+                "errorCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Error Code"
+                },
+                "errorMessage": {
+                  "type": "string",
+                  "description": "Error Message"
+                }
+              }
+            }
+          },
+          "422": {
+            "description": "422",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "success": {
+                  "type": "boolean",
+                  "description": "success"
+                },
+                "errorCode": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "Error Code"
+                },
+                "errorMessage": {
+                  "type": "string",
+                  "description": "Error Message"
+                },
+                "errors": {
+                  "type": "object",
+                  "properties": {
+                    "orderid": {
+                      "type": "array",
+                      "items": {
+                        "type": "string"
+                      },
+                      "description": "Error list"
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Delete a file from the order",
+        "tags": [
+          "file"
+        ],
+        "description": "Deletes the given file from the order, prior to an order being submitted. If a source file ever needs to be cancelled after it’s been submitted as part of an order, ‘CANCEL FILE’ can be submitted as a comment in /orders/{orderid}/files/{fileid}}/comment. We also request that you separately contact your Acclaro project manager with the source file cancellation request.",
```

Comment:
Please review all descriptions, summaries, titles, etc. They need to be free from grammar and spelling errors.

Reply:
corrected